howto/spark/spark_mlrun_describe.ipynb

```diff
@@ -0,0 +1,1417 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Spark Job with MLRun\n",
+    "Basic read csv spark mlrun function."
```

Comment:
Since this is a `howto`, please start by sharing what it is you show how to do and the goal of the notebook.